main.py

```diff
@@ -87,6 +87,11 @@ def faq():
     return render_template("faq.html", maps_api_key=app_state["maps_api_key"])
 
 
+@app.route("/faqq")
+def faqq():
+    return render_template("faqq.html", maps_api_key=app_state["maps_api_key"])
+    
+
```

Comment:
Why faqq instead of replacing faq?

Reply:
I believe I added this as a comment in the pr -- "As a quick solution, created new 'faqq' page to avoid showing other options in nav. We can probably do this with logic instead (with a bit more time)."

Reply:
FAQ should not extend nav.html because when it does, the top links break (e.g. click NY > FAQ > StateAll (you get CA) > CountyAll (you get a 404))

Let's have FAQ extend nonav.html instead, and replace faq.html with the new changes.